EmployeeManagementSystem/Helpers/ConsoleLogger.cs

```diff
@@ -0,0 +1,34 @@
+using System;
+
+namespace EmployeeManagement;
+public class ConsoleLogger : ILogger
+{
+    public void LogInfo(string message)
+    {
+        Console.WriteLine(message);
+    }
+
+    public void LogError(string message)
+    {
+        SetConsoleColor(ConsoleColor.Red);
```

Comment:
still certain code is repeating. you can have another private function called LogMessage